public/styles/main.css

```diff
@@ -91,6 +95,11 @@ a.connections-table.icon {
   font-size: 2rem;
   padding: 0.5rem;
 }
+.header.small {
```

Comment:
I'm not sure I like how we're re-suing `small` here for heading and test. If there's some shared meaning for small (say `font-size: 0.8em`) then it should be defined in it's own small class. If small means something completely different in the two cases we should probably just have `heading-small` as a single class. I get you're just following patterns elsewhere, but this is the time these patterns become fixed so I'm being a bit strict